core/src/main/java/com/linecorp/armeria/server/docs/ServiceSpecification.java

```diff
@@ -190,4 +196,12 @@ public ServiceSpecification(Iterable<ServiceInfo> services,
     public List<HttpHeaders> exampleHttpHeaders() {
         return exampleHttpHeaders;
     }
+
+    /**
+     * Returns armeria version of the services in this specification.
```

Comment:
the Armeria,
Returns the Armeria version of this specification.

Reply:
Thank you for your recommendation.
I'll change it.